src/components/GlobalLearningResourcesPage/GlobalLearningResourcesQuickstartItem.scss

```diff
@@ -30,5 +30,15 @@
 
   &--footer {
     margin-top: auto;
+
+    &-span {
+      margin-right: 10px;
+    }
+
+    &-icon {
+      width: 16px;
```

Comment:
Use PF variables, please. For the spacing you can even use PF class names directly on the element(s) if you prefer that.